src/xmpp_wf/get/xmpp_iq_send_invitation.c

```diff
@@ -52,6 +53,8 @@ static void xmpp_iq_send_invitation_cb(const char *msg,
         int code = get_info_int(msg, "code='", "'", NULL);
         int custom_code = get_info_int(msg, "custom_code='", "'", NULL);
 
+        error_code = custom_code;
```

Comment:
`custom_code `may be 0. E.g. "QoS limit reached" has only `code`.
I suggest to set it to `code` when `custom_code` is 0.